### ISSUE TITLE
panic: interface conversion: interface {} is **ec2.Instance, not *ec2.Instance

### ISSUE THREAD

**Issue by nicocesar** [mitchellh/packer#3588]
this happened only ONCE.  when I tryed to reproduce it with PACKER_LOG=1  it was ok. 

packer: 0.10.1
platform: Debian 3.2.65-1+deb7u1

```
2016/06/01 14:58:33 ui: ==> amazon-ebs: Waiting for SSH to become available...
2016/06/01 14:58:35 packer: panic: interface conversion: interface {} is **ec2.Instance, not *ec2.Instance
2016/06/01 14:58:35 packer: 
2016/06/01 14:58:35 packer: goroutine 101 [running]:
2016/06/01 14:58:35 packer: panic(0x12d6880, 0xc820383740)
2016/06/01 14:58:35 packer:     /usr/local/go/src/runtime/panic.go:481 +0x3e6
2016/06/01 14:58:35 packer: github.com/mitchellh/packer/builder/amazon/common.SSHHost.func1(0x7f2fa74ca978, 0xc820320030, 0x0, 0x0, 0x0, 0x0)
2016/06/01 14:58:35 packer:     /Users/cbednarski/go/src/github.com/mitchellh/packer/builder/amazon/common/ssh.go:19 +0x70d
2016/06/01 14:58:35 packer: github.com/mitchellh/packer/helper/communicator.(*StepConnectSSH).waitForSSH(0xc8203f21e0, 0x7f2fa74ca978, 0xc820320030, 0xc8202d5200, 0x0, 0x0, 0x0, 0x0)
2016/06/01 14:58:35 packer:     /Users/cbednarski/go/src/github.com/mitchellh/packer/helper/communicator/step_connect_ssh.go:119 +0x6e9
2016/06/01 14:58:35 packer: github.com/mitchellh/packer/helper/communicator.(*StepConnectSSH).Run.func1(0x7f2fa74ca418, 0xc82023b220, 0xc820445f10, 0xc820445f40, 0xc8203f21e0, 0x7f2fa74ca978, 0xc820320030, 0xc8202d5200, 0xc8201ec620)
2016/06/01 14:58:35 packer:     /Users/cbednarski/go/src/github.com/mitchellh/packer/helper/communicator/step_connect_ssh.go:39 +0x78
2016/06/01 14:58:35 packer: created by github.com/mitchellh/packer/helper/communicator.(*StepConnectSSH).Run
2016/06/01 14:58:35 packer:     /Users/cbednarski/go/src/github.com/mitchellh/packer/helper/communicator/step_connect_ssh.go:41 +0x1c0
2016/06/01 14:58:35 /root/packer/packer: plugin process exited
2016/06/01 14:58:35 ui error: Build 'amazon-ebs' errored: unexpected EOF
2016/06/01 14:58:35 Builds completed. Waiting on interrupt barrier...
2016/06/01 14:58:35 machine readable: error-count []string{"1"}
2016/06/01 14:58:35 ui error: 
==> Some builds didn't complete successfully and had errors:
2016/06/01 14:58:35 machine readable: amazon-ebs,error []string{"unexpected EOF"}
2016/06/01 14:58:35 ui error: --> amazon-ebs: unexpected EOF
2016/06/01 14:58:35 ui: 
==> Builds finished but no artifacts were created.
2016/06/01 14:58:35 waiting for all plugin processes to complete...

```

**Comment by dpiddy** [comment]
Oops, dup of https://github.com/mitchellh/packer/issues/3496?

**Comment by dpiddy** [comment]
Dup of https://github.com/mitchellh/packer/issues/3496?

**Comment by rickard-von-essen** [comment]
Yes dup. Closing in favour of #3496
